backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -59,8 +66,162 @@ def test_start_project(self):
+        self.assertEqual(LogEntry.objects.count(), 1)
+
+        card.status = AgileCard.REVIEW_FEEDBACK
+        card.save()
+        creators.log_card_moved_to_review_feedback(card, actor_user)
+        self.assertTrue(card.status == AgileCard.REVIEW_FEEDBACK)
+        self.assertEqual(LogEntry.objects.count(), 2)
+
+        card.status = AgileCard.COMPLETE
+        card.save()
+        creators.log_card_moved_to_complete(card, actor_user)
+        self.assertTrue(card.status == AgileCard.COMPLETE)
+        log_entries = [entry.event_type.name for entry in LogEntry.objects.all()].count('CARD_MOVED_TO_COMPLETE')
+        self.assertTrue(log_entries, 1)
+
+    @patch('django.utils.timezone.now',
+           return_value=datetime.datetime(
+            timezone.now().year,
+            timezone.now().month,
+            timezone.now().day,
+            timezone.now().hour,
+            timezone.now().minute,
+            timezone.now().second,
+            tzinfo=pytz.timezone('utc'))
+    )
+    def test_card_to_complete_then_rf_then_complete_creates_two_complete_entries_after_debounce_period(self, mock_now):
+
+        actor_user = UserFactory(is_superuser=True)
+        card = factories.AgileCardFactory(
+            status=AgileCard.IN_REVIEW,
+            content_item=factories.ProjectContentItemFactory(
+                project_submission_type=ContentItem.LINK, template_repo=None
+            ),
+        )
+
+        initial_complete_review = factories.RecruitProjectReviewFactory(
+            recruit_project=card.recruit_project,
+            timestamp=timezone.now(),
+            reviewer_user=actor_user
+        )
+        card.status = AgileCard.COMPLETE
+        card.save()
+        creators.log_card_moved_to_complete(card, actor_user)
+        self.assertTrue(card.status == AgileCard.COMPLETE)
+        self.assertEqual(LogEntry.objects.count(), 1)
+
+        nyc_review = factories.RecruitProjectReviewFactory(
+            recruit_project=card.recruit_project,
+            timestamp=initial_complete_review.timestamp + timezone.timedelta(seconds=20),
+            reviewer_user=actor_user
+        )
+        card.status = AgileCard.REVIEW_FEEDBACK
+        card.save()
+        creators.log_card_moved_to_review_feedback(card, actor_user)
+
+        complete_review = factories.RecruitProjectReviewFactory(
+            recruit_project=card.recruit_project,
+            timestamp=nyc_review.timestamp + timezone.timedelta(seconds=120),
+            reviewer_user=actor_user
+        )
+        card.status = AgileCard.COMPLETE
+        card.save()
+
+        mock_now.return_value = timezone.now() + timezone.timedelta(seconds=121)
+        creators.log_card_moved_to_complete(card, actor_user)
+        self.assertTrue(card.status == AgileCard.COMPLETE)
+
+        log_entries = [entry.event_type.name for entry in LogEntry.objects.all()].count('CARD_MOVED_TO_COMPLETE')
+        self.assertTrue(log_entries, 3)
```

Comment:
There should be 3 entries because the test starts with a review to complete status then review feedback status and then complete status again.  The 2nd complete status occurs after the debounce period of 120 seconds and therefore 3 entries should be created.